addons/utm/models/utm_mixin.py

```diff
@@ -25,23 +31,29 @@ def default_get(self, fields):
         if not self.env.is_superuser() and self.env.user.has_group('sales_team.group_sale_salesman'):
             return values
 
+        values.update(self._get_missing_utm_values(values))
+        return values
+
+    def _get_missing_utm_values(self, values, allow_create=False):
```

Comment:
For naming purpose, I would call the allow create parameter force_create . It is not really allowing.